Client/Frontend/Widgets/TwoLineCell.swift

```diff
@@ -4,11 +4,14 @@
 
 import UIKit
 
-private let ImageSize: CGFloat = 24
-private let ImageMargin: CGFloat = 12
-private let TextColor = UIAccessibilityDarkerSystemColorsEnabled() ? UIColor.blackColor() : UIColor(rgb: 0x333333)
-private let DetailTextColor = UIAccessibilityDarkerSystemColorsEnabled() ? UIColor.darkGrayColor() : UIColor.grayColor()
-private let DetailTextTopMargin = CGFloat(5)
+struct TwoLineCellUX {
+    static let ImageSize: CGFloat = 25
+    static let BorderViewMargin: CGFloat = 13
+    static let BorderWidth: CGFloat = 32
+    static let TextColor = UIAccessibilityDarkerSystemColorsEnabled() ? UIColor.blackColor() : UIColor(rgb: 0x333333)
+    static let DetailTextColor = UIAccessibilityDarkerSystemColorsEnabled() ? UIColor.darkGrayColor() : UIColor.grayColor()
+    static let DetailTextTopMargin = CGFloat(5)
```

Comment:
Nit: For the above constants you declare the type `CGFloat` but here you wrap the value `CGValue(5)`. I would just stick with on way of doing it for consistency.